src/tests/testHttpRequest.cc

```diff
@@ -16,6 +16,23 @@
 
 #include <cppunit/TestAssert.h>
 
+// XXX: cannot use stub_access_log.cc because it conflicts with log/liblog.la
+#include "PingData.h"
+ping_data::ping_data() :
+    n_sent(0),
+    n_recv(0),
+    n_replies_expected(0),
+    timeout(0),
+    timedout(0),
+    w_rtt(0),
+    p_rtt(0)
+{
+    start.tv_sec = 0;
+    start.tv_usec = 0;
+    stop.tv_sec = 0;
+    stop.tv_usec = 0;
+}
```

Comment:
This is a liblog.a dependency and we cannot remove this:

```
log/.libs/liblog.a(access_log.o): In function `HierarchyLogEntry::HierarchyLogEntry()':
/home/ed/work/squid/src/log/access_log.cc:178: undefined reference to `ping_data::ping_data()
```
I think we should add this into a separate stub (moving it also from stub_access_log.cc).